content/miq_dialogs/miq_provision_openstack_dialogs_template.yaml

```diff
@@ -164,6 +164,15 @@
           :required: true
           :display: :edit
           :data_type: :integer
+        :cloud_network_selection_method:
+          :values:
+            network: Cloud Network
+            port: Network Port
+          :description: How to Specify Network
```

Comment:
@sseago Can you suggest an alternative description for this field?  It stood out to me when reviewing the screenshot in https://github.com/ManageIQ/manageiq-ui-classic/pull/5103 that it does not match the feel of any other field description.

Would this be better worded as: "Network Type" or "Network Selection Type"

Reply:
@gmcculloug You're right -- it doesn't read very smoothly, although I think introducing "Type" into the label would make the wording worse, since it would imply that the selection would influence the actual object type of the resulting association. The wording there now is clear, but perhaps a bit clumsy. The back end param has been renamed cloud_network_selection_method -- maybe we could just use something based on this? I could change it to "Network Selection Method" -- i.e. with the choices of choose by "Cloud Network" vs choose by "Network Port", I think it would be as clear as the current wording, but less clumsy than the "How to..." verbiage.